web/beacon-app/src/features/topics/components/TopicQueryResult/QueryResultContent.tsx

```diff
@@ -0,0 +1,27 @@
+import { t } from '@lingui/macro';
+import React from 'react';
+
+import DisplayResultData from './DisplayResultData';
+interface QueryResultContentProps {
+  result: any;
+  mimeType: string;
+}
+
+const QueryResultContent: React.FC<QueryResultContentProps> = ({ result, mimeType }) => {
+  console.log('[] result', result); // added this avoid eslint error
+  console.log('[] mimetype', mimeType);
+
+  return (
+    <div className="shadow-md h-20 max-h-80 overflow-y-auto bg-black p-4 text-white">
+      <pre className="mx-auto text-sm">
```

Comment:
The text is difficult for me to read on the screen. Could the base font be used instead?

Reply:
Sure we could try the base font for now. with the real data, we could see if it's the best size.